internal/bundler/jazzer_bundler.go

```diff
@@ -503,3 +511,48 @@ func getUniqueArtifactName(dependency string, artifactsMap map[string]uint) stri
 
 	return newBaseName
 }
+
+func listFuzzTests(classNames []string, runtimeDeps []string) ([]string, error) {
+	listFuzzTestsJar, err := runfiles.Finder.ListFuzzTestsJarPath()
+	if err != nil {
+		return nil, err
+	}
+	classPath := strings.Join(append(runtimeDeps, listFuzzTestsJar), string(os.PathListSeparator))
+
+	args := []string{
+		"-cp",
+		classPath,
+		"com.code_intelligence.cifuzz.helper.ListFuzzTests",
+	}
+	if len(classNames) > 0 {
+		args = append(args, strings.Join(classNames, " "))
+	}
+
+	javaHome, err := runfiles.Finder.JavaHomePath()
+	if err != nil {
+		return nil, err
+	}
+	javaBin := filepath.Join(javaHome, "bin", "java")
+	if runtime.GOOS == "windows" {
+		javaBin = filepath.Join(javaHome, "bin", "java.exe")
+	}
+
+	buffer := new(bytes.Buffer)
+	cmd := exec.Command(javaBin, args...)
+	cmd.Stdout = buffer
+	cmd.Stderr = os.Stderr
+	log.Debugf("Command: %s", cmd.String())
+	err = cmd.Run()
+	if err != nil {
+		return nil, cmdutils.WrapExecError(errors.WithStack(err), cmd)
+	}
+
+	output, err := io.ReadAll(buffer)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
```

Comment:
This can be simplified:

```suggestion
cmd := exec.Command(javaBin, args...)
	cmd.Stderr = os.Stderr
	log.Debugf("Command: %s", cmd.String())
	output, err := cmd.Output()
	if err != nil {
		return nil, cmdutils.WrapExecError(errors.WithStack(err), cmd)
	}
```